méli-mélo/th-empathy/empathy.html

```diff
@@ -92,3 +92,111 @@ <h4>Code</h4>
 <pre><code>&lt;div class="wb-frmvld provisional wb-steps quiz empathy panel-body"&gt;
     &lt;!-- Steps Quiz Form --&gt;
 &lt;/div&gt;</code></pre>
+
+<h3>Life Journey Home Page Layout</h3>
+<div class="mrgn-tp-lg container">
```

Comment:
Either remove this container or use the page layout without container and add containers where needed.

```suggestion
<div class="mrgn-tp-lg">
```

Reply:
Comment addressed